app/components/Common/MapMarking/index.js

```diff
@@ -415,7 +415,7 @@ export default function MapMarking({
           setAccuracyInMeters(position.coords.accuracy);
           onUpdateUserLocation(position);
           setLocation(position);
```

Comment:
Do you think setting the retrieved location in the state variable is happening async and thus location can still be undefined above? Thus returning it as a local variable makes the information available at once?

Reply:
Answered already above ;-)

Reply:
Yes. You framed it in a sophisticated way.